vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -217,6 +226,12 @@ def parse_endpoint(entity)
       new_result
     end
 
+    def parse_namespaces(container_projects)
+      new_result = parse_base_item(container_projects)
+      new_result.except!(:namespace)
+      new_result
```

Comment:
Maybe we can have a one-liner?

    def parse_namespaces(container_projects)
      parse_base_item(container_projects).except(:namespace)
    end